gix-diff/tests/tree/mod.rs

```diff
@@ -253,17 +253,8 @@ mod changes {
                 ":100644 100644 28ce6a8b26aa170e1de65536fe8abe1832bd3242 13c2aca72ab576cb5f22dc8e7f8ba8ddab553a8a M	f/f"
             );
 
-            #[cfg(windows)]
-            let tree_with_link_id = hex_to_id("3b287f8730c81d0b763c2d294618a5e32b67b4f8");
-            #[cfg(windows)]
-            let link_entry_oid = hex_to_id("e69de29bb2d1d6434b8b29ae775ad8c2e48c5391");
-            #[cfg(windows)]
-            let link_entry_mode = EntryKind::Blob;
-            #[cfg(not(windows))]
             let tree_with_link_id = hex_to_id("7e26dba59b6336f87d1d4ae3505a2da302b91c76");
-            #[cfg(not(windows))]
             let link_entry_oid = hex_to_id("2e65efe2a145dda7ee51d1741299f848e5bf752e");
-            #[cfg(not(windows))]
             let link_entry_mode = EntryKind::Link;
```

Comment:
As detailed in #1442 and mentioned in https://github.com/Byron/gitoxide/issues/1440#issuecomment-2209619536, this changes the expected hash values to what I believe they *should* be on all platforms including Windows, but currently the repository that is created when the fixture script is actually run on Windows produces the situation described by the old values.

This comment is for reference in connection with #1442. I do not advocate the old code here be brought back.

Reply:
I see, thank you! I yet have to catch up on the new issues though, to see if there is anything that can be done.